src/hooks/useChats.js

```diff
@@ -10,7 +10,7 @@ import { logError } from '../helpers'
 export const useLatestMessages = ({ packageId }) => {
   const [LatestMessages, setLatestMessages] = useState([])
   const { data, loading } = useSubscription(GET_CHAT_MESSAGES_SUBSCRIPTION, {
-    variables: { package_id: packageId },
+    variables: { 'package_id': packageId },
```

Comment:
Why the single quotes are required? Or is a lint fix?

Reply:
I addressed that issue